### PR TITLE
Feature/lab08

### PR DESCRIPTION
**Pull request by Mapirlet**

### CODE REVIEW

**Comment by Mapirlet** [comment]
- Not sure about the explanation of the COPY line 426 of the README.md in labs/08_docker_compose.
- Not sure about the explanation of the docker-compose down

File: labs/08_docker_compose/README.md [hunk 1]
```diff
@@ -0,0 +1,820 @@
...
+
+When you run `docker-compose up`, Docker Compose reads the `docker-compose.yml` file and creates the services defined in the file. It then starts the services in the correct order, based on their dependencies.
+
+## 1. Prerequisites
+
+Before you start this lab, you should:
+
+- Be familiar with Docker containers and images.
+- Have Docker installed on your machine (Docker Compose is included with Docker Desktop for Windows and macOS).
+- Have a basic understanding of Docker Compose.
```
Comment (ThomasVrancken) [inline labs/08_docker_compose/README.md:35]: T'as rajouté une partie theorique sur Docker compose? Bien de commencer avec 1-2 slides et une intro sur la solution, comment ca marche... de facons plus visuelle

File: labs/08_docker_compose/README.md [hunk 2]
```diff
@@ -0,0 +1,820 @@
...
+
+## 1.3. Services 
+
+In the `docker-compose.yml` file, you can define multiple services for your application. Each service is defined as a separate section in the file. You can specify the image to be used or custom Dockerfiles to build the image, ports to expose, volumes to mount, environment variables, and other settings for each service.
+
+- **Service** : A service is a container that runs a specific application or process. For example, in a machine learning pipeline, you might have a data processing service, a model training service, and a model serving service.
+
+- **Build** : The build configuration allows you to build a custom image for the service using a Dockerfile. You can specify the path to the Dockerfile and the context for the build. Here the `Dockerfile` and the `docker-compose.yml` file are in the same directory, so we use `.` for the build path.
+
+- **Image** : The image to use for the service. You can use an existing image from Docker Hub or a custom image built from a Dockerfile. If we our machine learning pipeline to use a pre-built image from Docker Hub for the data processing service we could write in the `docker-compose.yml` file:
```
Comment (ThomasVrancken) [inline labs/08_docker_compose/README.md:140]: Remove the "we" in `If we our machine learning pipeline to use`

File: labs/08_docker_compose/deep_app/services/web-application/app.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import requests
+import sys
+import torch
+
+from flask import Flask, request, jsonify, render_template
+from subprocess import run
+
+# adding Folder_2 to the system path
```
Comment (ThomasVrancken) [inline labs/08_docker_compose/deep_app/services/web-application/app.py:8]: To be removed? Or I'd change the structure to not have to to use sys for this

File: labs/08_docker_compose/deep_app/services/web-application/templates/index.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
```
Comment (ThomasVrancken) [inline labs/08_docker_compose/deep_app/services/web-application/templates/index.html:1]: Nice one

**Review by ThomasVrancken — APPROVED** [review]
Tu peux la merge quand t'as review les qq minors comments que j'ai mis :)

**Pull request merged**